src/hostapi/coreaudio/pa_mac_core.c

```diff
@@ -2646,20 +2665,20 @@ static PaError CloseStream( PaStream* s )
        if( stream->outputUnit ) {
           int count = removeFromXRunListenerList( stream );
           if( count == 0 )
-             AudioDeviceRemovePropertyListener( stream->outputDevice,
+             PaMacCore_AudioDeviceRemovePropertyListener( stream->outputDevice,
                                                 0,
                                                 false,
                                                 kAudioDeviceProcessorOverload,
-                                                xrunCallback );
+                                                xrunCallback, NULL ); //todo: do we need to pass actual node?
```

Comment:
What is the status of this TODO? it would be better to resolve this before merging the PR